src/routes/dashboard/+layout.svelte

```diff
@@ -0,0 +1,8 @@
+<script>
```

Comment:
I think we moved away from dashboard layout and moved all the sign out and rerouting logic to the main layout. @niccoloMG can you confirm?

Reply:
Yep. dashboard/layout file isn't needed anymore. Conditional sign out button render and rerouting happens in the main layout